docs/topics/mta-rn-upgrade-ui.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-rn-upgrade-ui_{context}"]
+= Upgrading the User interface
+
+When you update the CLI of {ProductName} ({ProductShortName}) from version 6.0.0 to version {ProductVersion}, be sure to set the correct value of `rwx_supported`.
+
+.Procedure
+
+. Follow steps 1-8 of the installation instructions in the link:{ProductDocUserInterfaceGuideURL}/mta-6-installing-web-console-on-openshift_user-interface-guide#installing_the_migration_toolkit_for_applications_operator_and_the_user_interface:[user interface guide].
+. Click *YAML* view and make any needed changes to the CR settings.
+. {ProductShortName} {ProductVersion} includes a new parameter, `rwx_supported` that you must add to the CR settings in *YAML* view. This parameter determines who can run {ProductShortName}.
+
+** If you set it to `true`, only users with RWX permissions can run {ProductShortName}.
+** If you set it to `false`, other users can run {ProductShortName}.
```

Comment:
Same here, this is not an obligation to configure this parameter. If user will not update tackle CR with it - it will just work same way as if it was set to true. It is obligatory to configure it only if user wants to disable it.